src/appkit.cs

```diff
@@ -13334,7 +13639,7 @@ interface NSSliderCell {
 	[Mac (10,12,2)]
 	[BaseType (typeof(NSTouchBarItem))]
 	[DisableDefaultCtor]
-	interface NSSliderTouchBarItem
+	interface NSSliderTouchBarItem : INSUserInterfaceCompression
```

Comment:
You want to use `NSUserInterfaceCompression` instead or generator will pick the wrong interface.